Packages/com.unity.inputsystem/Documentation~/UISupport.md

```diff
@@ -16,26 +16,80 @@ You can use the following properties to configure [`InputSystemUIInputModule`](.
+
+#### Navigation-type input
+
+Navigation-type input controls the current selection based on motion read from the [`move`](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html#UnityEngine_InputSystem_UI_InputSystemUIInputModule_move) action. Additionally, input from
+[`submit`](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html#UnityEngine_InputSystem_UI_InputSystemUIInputModule_submit) will trigger `ISubmitHandler` on the currently selected object and
+[`Cancel`](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html#UnityEngine_InputSystem_UI_InputSystemUIInputModule_cancel) will trigger `ICancelHandler` on it.
+
+Inputs received on these actions is independent of each other. Thus, an arbitrary set of devices can be bound to these actions and feed input in arbitrary order.
+
+Navigation input is non-positional, i.e. unlike with pointer-type input, there is no screen position associcated with these actions. Rather, navigation actions always operate on the current selection.
+
+#### Tracked-type input
+
+Input from [tracked devices](../api/UnityEngine.InputSystem.TrackedDevice.html) such as [XR controllers](../api/UnityEngine.InputSystem.XR.XRController.html) and [HMDs](../api/UnityEngine.InputSystem.XR.XRHMD.html) essentially behaves like [pointer-type input](#pointer-type-input). The main difference is that the world-space device position and orientation sourced from  [`trackedDevicePosition`](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html#UnityEngine_InputSystem_UI_InputSystemUIInputModule_trackedDevicePosition) and  [`trackedDeviceOrientation`](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html#UnityEngine_InputSystem_UI_InputSystemUIInputModule_trackedDeviceOrientation) is translated into a screen-space position via raycasting.
+
+For this raycasting to work, you need to add [`TrackedDeviceRaycaster`](../api/UnityEngine.InputSystem.UI.TrackedDeviceRaycaster.html) to the `GameObject` that has the UI's `Canvas` component. This `GameObject` will usually have a `GraphicRaycaster` component which, however, works for 2D screen-space raycasting only. You can put [`TrackedDeviceRaycaster`](../api/UnityEngine.InputSystem.UI.TrackedDeviceRaycaster.html) alongside `GraphicRaycaster` and both can be enabled at the same time without advserse effect.
+
+![TrackedDeviceRayster Add Component](Images/TrackedDeviceRaycasterComponentMenu.png)
+
+![TrackedDeviceRayster Properties](Images/TrackedDeviceRaycaster.png)
+
+Note that clicks on tracked devices do not differ from other [pointer-type input](#pointer-type-input) meaning that actions such as [`Left Click`](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html#UnityEngine_InputSystem_UI_InputSystemUIInputModule_leftClick) work for tracked devices just like they work for other pointers.
```

Comment:
Split this sentence in two:

Clicks on tracked devices do not differ from other [pointer-type input](#pointer-type-input). Therefore, actions such as [`Left Click`](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html#UnityEngine_InputSystem_UI_InputSystemUIInputModule_leftClick) work for tracked devices just like they work for other pointers.

Reply:
Done